Geometry/TrackerGeometryBuilder/test/trackerParametersFromDD4hep_cfg.py

```diff
@@ -0,0 +1,20 @@
+import FWCore.ParameterSet.Config as cms
+
+process = cms.Process("TrackerParametersTest")
+process.load('Configuration.Geometry.GeometryExtended_cff')
+process.load('Configuration.Geometry.GeometryExtendedReco_cff')
+
+process.trackerGeometry.applyAlignment = cms.bool(False)
+
+process.source = cms.Source("EmptySource")
+
+process.maxEvents = cms.untracked.PSet(
+    input = cms.untracked.int32(1)
+)
+
+process.test = cms.EDAnalyzer("TrackerParametersAnalyzer")
+
+process.p1 = cms.Path(process.test)
+
```

Comment:
What part of this config is DD4hep? GeometryExtended is Run 1. Is the name of the file incorrect?

Reply:
No, the configuration should and has been changed locally. I've frozen this PR on request from @ghugo83 .